src/dialog_launcher.ts

```diff
@@ -143,6 +146,18 @@ export class Launcher extends search.Search {
                     b_name_weight = Math.min(b_name_weight, levenshtein.compare(pattern_lower, b.description.toLowerCase()))
                 }
 
+                if (this.mru_list) {
+                    const a_recent = this.mru_list.is_recent(a);
```

Comment:
Since the underlying list is actually in an order based on most recently used, maybe instead of a "bool" I should return the index so we can rank based on that?